packages/bitcore-node/src/models/coin.ts

```diff
@@ -187,6 +189,27 @@ export class CoinModel extends BaseModel<ICoin> {
   }
 
   _apiTransform(coin: Partial<MongoBound<ICoin>>, options?: { object: boolean }): any {
+    // try to parse coin.address if its 'false' and script exists
+    if (coin.address == 'false' && coin.script != undefined && coin.script != Buffer.alloc(0)) {
```

Comment:
coin.script will always != Buffer.alloc(0) because they will be different instances of the prototypes. For example, `Buffer.alloc(0) == Buffer.alloc(0)` is false. 

I think this should be `coin.script.toString() != '')`

Reply:
Changed in latest commit.